### PR TITLE
add a dialog that shows all available shortkeys

### PR DESCRIPTION
**Pull request by thomasheyenbrock**
This adds a dialog that shows all the current short keys.

CleanShot 2022-07-29 at 16 20 55@2x

### CODE REVIEW

File: packages/graphiql/src/components/GraphiQL.tsx [hunk 1]
```diff
@@ -997,6 +999,62 @@ class GraphiQLWithContext extends React.Component<
...
+          <div className="graphiql-dialog-header">
+            <div className="graphiql-dialog-title">Short Keys</div>
+            <Dialog.Close
+              onClick={() => {
+                this.setState({ showShortKeys: false });
+              }}
+            />
+          </div>
+          <div className="graphiql-dialog-section">
+            <ul className="graphiql-short-keys">
```
Comment (acao) [inline packages/graphiql/src/components/GraphiQL.tsx:1016]: Do you plan to move this to JSON in later iterations?

Comment (acao) [inline packages/graphiql/src/components/GraphiQL.tsx:1016]: I'm guessing you plan on moving this to JSON eventually?

Also, you could add a little note on this dialog if you want that contains a little link to these docs, keyed by `prop.keyMap`:
https://codemirror.net/5/demo/sublime.html
https://codemirror.net/5/demo/vim.html

also, just noticed i was completely wrong about that vim PR! this does have insert mode already, i forgot, this should be good to say we already have vim support. i will correct myself there.

**Review by acao — APPROVED** [review]
Looks great to me! And the design system appears to be working out wonderfully here! Need another next release? I'm going to suggest an improvement as a new issue to not slow you down!

**Comment by acao** [comment]
@thomasheyenbrock another thought for later perhaps, what if this were an html table or used table cell layout or similar to achieve alignment of the right-hand entries?

**Comment by thomasheyenbrock** [comment]
Nice suggestions @acao! Here's the second iteration:
CleanShot 2022-08-01 at 19 08 52@2x

**Pull request merged**